CMakeModules/FindCBLAS.cmake

```diff
@@ -319,7 +319,7 @@ IF(NOT CBLAS_LIBRARIES)
   CHECK_ALL_LIBRARIES(
     CBLAS_LIBRARIES
     CBLAS
-    cblas_dgemm
+    dgemm_
```

Comment:
I've used blas from lapacke from your windows dependencies archive,
cblas_dgemm seems not to be present there.

On Mon, Aug 29, 2016 at 7:51 PM, Pavan Yalamanchili <
notifications@github.com> wrote:

> In CMakeModules/FindCBLAS.cmake
> <https://github.com/arrayfire/arrayfire/pull/1554#discussion_r76723440>:
>
> > @@ -319,7 +319,7 @@ IF(NOT CBLAS_LIBRARIES)
> >    CHECK_ALL_LIBRARIES(
> >      CBLAS_LIBRARIES
> >      CBLAS
> > -    cblas_dgemm
> > +    dgemm_
>
> Why is this necessary ? In what situations is cblas.h present but
> cblas_dgemm absent?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/arrayfire/arrayfire/pull/1554/files/bce7db5b959644de84c343eb41f3bb22ff57d332#r76723440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACSyAGJ89vKIrri4xaUS6J5Ymod9Ewjjks5qk5rMgaJpZM4Jvtab>
> .
>